tests/test_transforms/test_encoders/test_mean_encoder_transform.py

```diff
@@ -0,0 +1,250 @@
+import numpy as np
```

Comment:
It seems like we haven't tried different values of smoothing here.

Reply:
All smoke tests use it

Reply:
We don't check that it changes the values.

Reply:
Fixed